badges/badges.css

```diff
@@ -0,0 +1,41 @@
+.disp-badges {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 1rem;
+}
+
+.disp-badge-item {
+  position: relative;
+  width: 100px;
+}
+
+.badge-text {
+  text-align: center;
+  margin: 0;
+  /* width: 80px; */
+}
+
+.sub-title {
+  font-size: 1.2rem;
+}
+
+.avatar {
+  border-radius: 50%;
+  height: 100px;
+  width: 100px;
```

Comment:
if possible maintain consistency in units.

Reply:
Done, man!